generate/genhcl/genhcl_test.go

```diff
@@ -1207,10 +1249,31 @@ func TestLoadGeneratedHCL(t *testing.T) {
 					),
 				},
 			},
-			wantErr: errors.E(genhcl.ErrLabelConflict),
+			want: []result{
```

Comment:
The test name was not updated [here](https://github.com/mineiros-io/terramate/pull/393/files#diff-74888afee21574583ccb4e0af6e3ab123ccf5cc5dfb2715c9cd156539a987ce6R1227), it still says that blocks with same label fails.

Reply:
fixed, thanks